stable/mariadb/templates/slave-statefulset.yaml

```
@@ -74,6 +74,10 @@ spec:
 {{ toYaml . | indent 8 }}
       {{- end }}
       {{- end }}
+      {{- if .Values.slave.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.slave.nodeSelector | indent 8 }}
```

can you use `nindent` instead of `indent`? That way you can tab the toYaml line and it is easier to read.

```suggestion
        {{ toYaml .Values.slave.nodeSelector | nindent 8 }}
```

---

I was following the practice in the charts. I don't see this used anywhere else in this chart. Do you want this to be changed in both the master-statefulset.yaml and slave-statefulset.yaml?

---

I changed this in the master-statefulset.yaml and slave-statefulset.yaml